tests/test_parser.py

```diff
@@ -0,0 +1,132 @@
+import unittest
+import warnings
+from pathlib import Path
+
+import rdflib
+
+_TEST_DIR = Path(__file__).resolve().parent
+
+
+class TestGraphParsing(unittest.TestCase):
+    def test_parsing_ox_turtle_bulk_load(self):
+        graph = rdflib.Graph(store="Oxigraph")
+        graph.parse(_TEST_DIR / "data/test.ttl", format="ox-turtle", transactional=False)
+        self.assertEqual(len(graph), 6)
+
+    def test_parsing_ox_turtle_load(self):
+        graph = rdflib.Graph(store="Oxigraph")
+        graph.parse(_TEST_DIR / "data/test.ttl", format="ox-turtle", transactional=True)
+
+        self.assertEqual(len(graph), 6)
+
+    def test_parsing_ox_turtle_fallback(self):
+        graph = rdflib.Graph()
+        with warnings.catch_warnings(record=True) as warning:
+            graph.parse(_TEST_DIR / "data/test.ttl", format="ox-turtle", transactional=False)
+
+        self.assertEqual(
+            warning[0].message.args[0],
+            (
+                "Graph store should be an instance of OxigraphStore, got Memory"
+                " store instead. Attempting to parse using rdflib native parser."
+            ),
+        )
+        self.assertEqual(len(graph), 6)
+
+    def test_parsing_ox_url_turtle(self):
+        graph = rdflib.Graph(store="Oxigraph")
+        graph.parse(
+            "https://i-adopt.github.io/ontology/ontology.ttl",
+            format="ox-turtle",
+            transactional=True,
+        )
+        self.assertIsNotNone(graph)
+
+    def test_parsing_ox_ntriples_bulk_load(self):
+        graph = rdflib.Graph(store="Oxigraph")
+        graph.parse(_TEST_DIR / "data/test.nt", format="ox-ntriples", transactional=False)
```

Comment:
sadly data/test.nt is missing

Reply:
my bad, used `git commit -am "..."` , instead of `git add ...` so file renaming did not end up commited. fixed now